content/about/notes/1.1/index.md

```diff
@@ -113,7 +113,8 @@ a richer visualization experience. See the [Kiali task](/docs/tasks/telemetry/ki
     - Improved the protocol between Envoy and Mixer.
 
 - **Control Headers and Routing**. Added the option to create adapters to influence
-an incoming request's headers and routing.
+an incoming request's headers and routing. See [Control Headers and Routing task](/docs/tasks/policy-enforcement/control-headers)
```

Comment:
See -> See the
Put the word "task" outside the link.

Reply:
Done